cluster/mesos/docker/deployAddons.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# deploy the add-on services after the cluster is available
+
+set -e
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../../..
+source "config-default.sh"
+
+if [ "${ENABLE_CLUSTER_DNS}" == true ]; then
+	echo "Deploying DNS on kubernetes"
+	sed -e "s/{{ pillar\['dns_replicas'\] }}/${DNS_REPLICAS}/g;s/{{ pillar\['dns_domain'\] }}/${DNS_DOMAIN}/g" $KUBE_ROOT/cluster/addons/dns/skydns-rc.yaml.in > dns/skydns-rc.yaml
+	sed -e "s/{{ pillar\['dns_server'\] }}/${DNS_SERVER_IP}/g" $KUBE_ROOT/cluster/addons/dns/skydns-svc.yaml.in > dns/skydns-svc.yaml
+
+	KUBECONFIG=$("${KUBE_ROOT}/cluster/kubectl.sh" config view | base64)
+	sed -e "s,KUBECONFIG,$KUBECONFIG," dns/skydns-secret.yaml.in > dns/skydns-secret.yaml
```

Comment:
Does this actually work? When I do a kubectl config view it redacts the secrets...

```
$ kubectl config view
apiVersion: v1
clusters:
- cluster:
    server: http://172.17.0.60:8888
  name: mesos/docker
- cluster:
    certificate-authority-data: REDACTED
    server: https://10.245.1.2
  name: vagrant
contexts:
- context:
    cluster: mesos/docker
    user: mesos/docker
  name: mesos/docker
- context:
    cluster: vagrant
    user: vagrant
  name: vagrant
current-context: mesos/docker
kind: Config
preferences: {}
users:
- name: vagrant
  user:
    client-certificate-data: REDACTED
    client-key-data: REDACTED
    password: vagrant
    username: vagrant
```